core/src/test/java/tech/pegasys/web3signer/core/service/jsonrpc/EthAccountsResultProviderTest.java

```diff
@@ -22,30 +22,39 @@
 import tech.pegasys.web3signer.core.service.jsonrpc.handlers.internalresponse.EthAccountsResultProvider;
 import tech.pegasys.web3signer.signing.secp256k1.EthPublicKeyUtils;
 
+import java.security.GeneralSecurityException;
+import java.security.SecureRandom;
 import java.security.interfaces.ECPublicKey;
 import java.util.List;
 import java.util.Set;
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import com.google.common.collect.Sets;
-import org.apache.tuweni.bytes.Bytes;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.web3j.crypto.Keys;
 
-@SuppressWarnings("unchecked")
 public class EthAccountsResultProviderTest {
-
-  final ECPublicKey publicKeyA = createKeyFrom("A".repeat(128));
-  final ECPublicKey publicKeyB = createKeyFrom("B".repeat(128));
-  final ECPublicKey publicKeyC = createKeyFrom("C".repeat(128));
-
-  final String addressA = Keys.getAddress(EthPublicKeyUtils.toHexString(publicKeyA));
-  final String addressB = Keys.getAddress(EthPublicKeyUtils.toHexString(publicKeyB));
-  final String addressC = Keys.getAddress(EthPublicKeyUtils.toHexString(publicKeyC));
-
-  final ECPublicKey createKeyFrom(final String hexString) {
-    return EthPublicKeyUtils.createPublicKey(Bytes.fromHexString(hexString));
+  private ECPublicKey publicKeyA;
+  private ECPublicKey publicKeyB;
+  private ECPublicKey publicKeyC;
+
+  private String addressA;
+  private String addressB;
+  private String addressC;
+
+  @BeforeEach
+  public void init() throws GeneralSecurityException {
+    SecureRandom secureRandom = new SecureRandom();
```

Comment:
Is this going to cause an issue with entropy in CI creating this multiple times?

Reply:
probably it will, let me create a static instance.

Reply:
fixed.